tests/transports-integrations/tests/integrations/test_google.py

```
@@ -0,0 +1,472 @@
+
+    return google_tools
+
+
+def load_image_from_url(url: str):
+    """Load image from URL for Google GenAI"""
+    if url.startswith("data:image"):
+        # Base64 image
+        header, data = url.split(",", 1)
+        img_data = base64.b64decode(data)
+        return Image.open(io.BytesIO(img_data))
+    else:
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)